src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -892,6 +914,21 @@ public record SecretAddress<T>(Uri Url) : ISecret<T> {
 public record SecretData<T>(ISecret<T> Secret) {
 }
 
+public record JobResult(
+    [PartitionKey] Guid JobId,
+    [RowKey] string Project,
```

Comment:
I don't think there's ever a case where we can have one job for multiple projects right? I'm just thinking the `RowKey` might not be necessary but I don't know if there's any pros/cons to including it anyway.

Reply:
Correct. I suppose we could just double up the partition/row key on the job id?

Reply:
Yup, you could do `[PartitionKey][RowKey] Guid JobId` like we do here: https://github.com/microsoft/onefuzz/blob/8980988fe083cc1aa79b1895c6b40e97763a10e6/src/ApiService/ApiService/OneFuzzTypes/Model.cs#L728